<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resources</class>
 <widget class="QMainWindow" name="Resources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resources</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>540</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writingCenterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Writing Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computerLabsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Computer Labs (free printing available)</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="careerCenterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Career Center</string>
    </property>
    <property name="icon">
     <iconset theme="face-monkey">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>651</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Here you will find the different resources BMCC has to offer to students. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Please select one of the options below for more information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tutoringCenterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tutoring Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scholarshipsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Scholarships</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fitnessCenterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fitness Center</string>
    </property>
    <property name="icon">
     <iconset theme="face-monkey">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="counselingCenterButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Counseling Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="foodPantryButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Food Pantry</string>
    </property>
   </widget>
   <widget class="QPushButton" name="daycareButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Early Childhood Center</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>backButton</zorder>
   <zorder>writingCenterButton</zorder>
   <zorder>computerLabsButton</zorder>
   <zorder>careerCenterButton</zorder>
   <zorder>tutoringCenterButton</zorder>
   <zorder>scholarshipsButton</zorder>
   <zorder>fitnessCenterButton</zorder>
   <zorder>counselingCenterButton</zorder>
   <zorder>foodPantryButton</zorder>
   <zorder>daycareButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
